lindsey-paul-challenge4/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import TextField from 'material-ui/TextField';
+import injectTapEventPlugin from 'react-tap-event-plugin';
+import FlatButton from 'material-ui/FlatButton';
+
+//injectTapEventPlugin();
+
+var contactName;
+var description;
+var price;
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = ({
+      contactName: "",
+      description: "",
+      price: "",
+    })
+  }
+
+handleSubmit() {
+  contactName = this.state.contactName,
+  description = this.state.description,
+  price = this.state.price,
+  this.props.onSubmit({contactName: contactName, description: description, price: price});
```

Comment:
why do you set these variables instead of just doing `this.props.onSubmit({contactName: this.state.contactName, description: this.state.description, price: this.state.price});`?